docs/reference/indices/rollover-index.asciidoc

```diff
@@ -206,13 +211,69 @@ The API returns the following response:
 <2> Whether the rollover was dry run.
 <3> The result of each condition.
 
+
+[[rollover-data-stream-ex]]
+===== Roll over a data stream
+
+[source,console]
+--------------------------------------------------
+PUT _data_stream/my-data-stream <1>
+{
+  "timestamp_field": "@timestamp"
+}
+
+# Add > 1000 documents to my-data-stream
+
+POST /my-data-stream/_rollover <2>
+{
+  "conditions" : {
+  "max_age": "7d",
+  "max_docs": 1000,
+  "max_size": "5gb"
+  }
+}
+--------------------------------------------------
+<1> Creates a data stream called `my-data-stream` with one initial backing index
+named `my-data-stream-000001`.
+<2> If the current index of `my-data-stream` was created 7 or more days ago, or
+contains 1,000 or more documents, or has an index size at least around 5GB, then
+a new backing index `my-data-stream-000002` index is created and added to the
+`my-data-stream` data stream.
```

Comment:
The conditions become a bit hard to parse in a callout. I think breaking them out into a bulleted list might help>
```suggestion
<2> This request creates a new backing index, `my-data-stream-000002`, and adds
it as the write index for the `my-data-stream` data stream if the current
write index meets at least one of the following conditions:
+
--
* The index was created 7 or more days ago.
* The index has an index size of 5GB or greater.
* The index contains 1,000 or more documents.
--
```